collection-scripts/gather

```diff
@@ -51,7 +51,7 @@ all_ns_resources+=(leases)
 
 # Run the Collection of Resources using inspect
 # running across all-namespaces for the few "Autoscaler" resources.
-oc adm inspect --dest-dir must-gather --rotated-pod-logs "${named_resources[@]}" &
+oc adm inspect --since="${MUST_GATHER_SINCE}" --since-time="${MUST_GATHER_SINCE_TIME}" --dest-dir must-gather --rotated-pod-logs "${named_resources[@]}" &
```

Comment:
Should we handle the cases where `MUST_GATHER_SINCE` or `MUST_GATHER_SINCE_TIME` are unset such as?;

```sh
oc adm inspect --since= --since-time= --dest-dir must-gather
```

Reply:
When `--since` and `--since-time` are unset, `oc adm must-gather` will still set both envs with Go's empty value for each type, so if the client doesn't specify either flag they will be `0s` and an empty string respectively. My manual testing show that this works.
This approach isn't the most clear though... WDYT?

Reply:
Yeah, I agree that this isn't clear. Besides if there is any change in defaults, these commands would fail. I think, we can only pass these flags when they are set.

Reply:
Latest push addresses your comment @ardaguclu, PTAL.